app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -43,4 +46,18 @@ object FeatureFlags {
      * Enables picture-in-picture feature
      */
     val pictureInPicture = Config.channel.isNightlyOrDebug
+
+    /**
+     * Allows edit of saved logins.
+     */
+    val loginEdit = when {
```

Comment:
I might be misunderstanding but why can't we just use the convention of `Config.channel.isNightlyOrDebug` here?